scRNA-seq/post-processing_scRNA-seq.Rmd

```diff
@@ -0,0 +1,98 @@
+---
+title: "scRNA-seq_post_processing"
+output:   
+  html_notebook: 
+    toc: true
+    toc_float: true
+---
+
+**CCDL 2019**
+
+In this notebook, we'll perform quality control analyses and normalization of 
+scRNA-seq count data. 
+
+As opposed to bulk rna-seq, there are there are a few main things to look out for
+in single cell RNA-seq:
+
+**Single-cell rna-seq has...**  
+- More amplification (and more of it's associated biases and error)  
+- More 0's in the gene expression data (most genes aren't expressed across cell types)  
```

Comment:
```suggestion
- More zeroes in the gene expression data (most genes aren't expressed across cell types).
People sometimes refer to this as "sparsity" or "dropout."
```
Trying to introduce people to how researchers that are more methodologically inclined talk about different problems or phenomena wherever possible.